hops/constants.py

```diff
@@ -88,6 +88,7 @@ class SSL_CONFIG:
     K_CERTIFICATE_CONFIG = "k_certificate"
     T_CERTIFICATE_CONFIG = "t_certificate"
     PEM_CLIENT_CERTIFICATE_CONFIG = "client.pem"
+    PEM_DIR = "/tmp"
```

Comment:
Do not assume that the temporary directory will always be `/tmp` Use Python's build-in function https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir

Reply:
Also, why do you need to change the location? I don't know if there are other libraries that assume the extracted pem files are in the current working directory. Can you please verify with Moritz and Till?

Reply:
@gibchikafa have you confirmed with @moritzmeister and @tdoehmen that it does not affect hsfs or flyingduck?

Reply:
We use `hsfs.client._get_ca_chain_path()` etc in the flyingduck client. As long as that returns the correct path it should be fine